scripts/ci/build_emscripten.sh

```diff
@@ -71,8 +73,43 @@ make soljson
 
 cd ..
 mkdir -p upload
-cp "$BUILD_DIR/libsolc/soljson.js" upload/
-cp "$BUILD_DIR/libsolc/soljson.js" ./
+scripts/ci/pack_soljson.sh "$BUILD_DIR/libsolc/soljson.js" "$BUILD_DIR/libsolc/soljson.wasm" > upload/soljson.js
+node - "$BUILD_DIR/libsolc/soljson.wasm" << "EOF"
```

Comment:
I'll move these tests to a proper location - but it should probably test it already.